src/34_Command_Line_Interface.md

```diff
@@ -1,7 +1,7 @@
 # Command Line Interface
```

Comment:
Interesting, I can't seem to find this page in https://docs.gitpod.io (maybe it hasn't been deployed yet).

Reply:
Yes, it is not deployed